docs/Test.md

```diff
@@ -53,6 +53,9 @@ $user = test::double('User', ['tableName' => 'fake_users']);
 User::tableName(); // 'fake_users'
 $user->verifyInvoked('tableName'); // success
 
+# on a static property where the property is an object.
+User::$staticProperty = test::double('PropertyClass', ['Method'=>"return value"]);
```

Comment:
this isn't really a special case, doing `User::$staticProperty = …` isn't related to this library, can be done in pure php